src/assets/index.css

```diff
@@ -0,0 +1,157 @@
+body {
+  margin: 0;
+  font-family:
+    -apple-system,
+    BlinkMacSystemFont,
+    'Segoe UI',
+    'Roboto',
+    'Oxygen',
+    'Ubuntu',
+    'Cantarell',
+    'Fira Sans',
+    'Droid Sans',
+    'Helvetica Neue',
+    sans-serif;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+}
+
+code {
+  font-family:
+    source-code-pro,
+    Menlo,
+    Monaco,
+    Consolas,
+    'Courier New',
+    monospace;
+}
+
+.Navbar {
+  display: flex;
+  width: 100%;
+  justify-content: space-between;
+  padding-top: 10px;
+}
+
+.cities-table {
+  margin-left: 40px;
+  margin-right: 40px;
+  display: grid;
+  grid-template-columns: repeat(2, 50%);
+  height: 600px;
+  gap: 10px;
+  padding-right: 10px;
+}
+
+.link {
+  background-color: #386479;
+  text-decoration: none;
+  box-shadow: 5px 5px 10px 5px #386479;
+  border-radius: 6px;
+  border: 1px solid rgba(122, 121, 121, 0.575);
+}
+
+img {
+  display: block;
+  margin-right: auto;
+  margin-left: auto;
+  margin-top: 20px;
+  width: 80px;
+  border-radius: 30%;
+}
+
+svg {
+  color: white;
+}
+
+.logo {
+  color: white;
+  font-family: GillSansBold, sans-serif;
+  text-transform: uppercase;
+  margin-left: -120px;
+}
+
+.my-link {
+  display: flex;
+  justify-content: right;
+}
```

Comment:
I will suggest you implement this feature according to design:

Your Implementation
![](https://user-images.githubusercontent.com/51296741/149642031-816cf675-75c9-4de4-8b8f-44c5dd72c029.png)

Project design:

 ![](https://user-images.githubusercontent.com/51296741/149641940-d40aaacf-cd56-4f1b-a6b4-8fcfa6d8b33c.png)

Reply:
![image](https://user-images.githubusercontent.com/71874425/149654982-1e6a0101-6dd4-424a-a7c6-fb6de247443b.png)

Above is the right screenshot, and as you can see in my style, the justify-content is on the right position. you can check the whole project on the live demo link

Reply:
Good job if it's now fixed. The demo link was taking me to a 404 page during the time of review.